NGTrackForce/src/app/components/associate-list-page/associate-list-page.component.ts

```diff
@@ -57,6 +58,12 @@ export class AssociateListPageComponent implements OnInit, OnDestroy, AfterViewI
   constructor(private clientService: ClientService, public associateService: AssociateService, public dialog: MatDialog, public lsHelp: Helpers) { }
 
   ngOnInit() {
+   this.lsHelp.removeStorageItem("clientGetAll");
```

Comment:
not clear why this is being done

Reply:
local window storage needs global helper, this is done to fix permanent loading issue